enrich-points-with-boundaries/geodatabase/src/main/scala/com/precisely/bigdata/sample/spark/GeodatabaseEnrichment.scala

```diff
@@ -78,9 +80,24 @@ object GeodatabaseEnrichment {
       table.search(filterSearch)
     }
 
-    spark.read.option("delimiter", "\t").option("header", "true").csv(addressFabricPath)
+    val df = spark.read.option("delimiter", "\t").option("header", "true").csv(addressFabricPath)
       .select("PBKEY", "LON", "LAT")
       .withSpatialSearchColumns(Seq(col("LON").cast(DoubleType), col("LAT").cast(DoubleType)), contains, includeEmptySearchResults = false)
-      .write.mode(SaveMode.Overwrite).parquet(outputPath)
+
+    /*
+       LI API supports attributes converters to convert a geometry to multiple available formats e.g. WKT, WKB, GeoJSON,
+       GeoPackageGeometry and KML. LI API provides Utility classes to convert the geometry for all of them.
```

Comment:
If ok with you, We may extend this comment like ...
`LI API supports attributes converters to convert a geometry to multiple available formats e.g. WKT, WKB, GeoJSON, GeoPackageGeometry and KML. LI API provides Utility classes to convert the geometry for all of them. More converters can be found in 'com.mapinfo.midev.persistence' package of LISDK javadocs https://docs.precisely.com/docs/sftw/hadoop/landingpage/docs/li/lisdk/javadocs/index.html
`

Reply:
Done.